pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -35,7 +35,7 @@ type framework struct {
 	nodeInfoSnapshot *cache.NodeInfoSnapshot
 	waitingPods      *waitingPodsMap
 	plugins          map[string]Plugin // a map of initialized plugins. Plugin name:plugin instance.
-	queueSortPlugins []QueueSortPlugin
+	queueSortPlugin  QueueSortPlugin   // only one queue sort plugin may be enabled at a time.
```

Comment:
It is not necessary to replace the slice now, we may have a different approach in the future, please the comment [here](https://github.com/kubernetes/kubernetes/pull/77529#discussion_r284017463). We could have multiple queue sort plugins, and framework could call the next plugin when current one returns `equal`.

Reply:
Thanks for it! It is a little confusing that both [code](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/v1alpha1/framework.go#L199) and [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md#queue-sort) say that only permitting one queue sort. If we intend to support queue sort like the [comment](https://github.com/kubernetes/kubernetes/pull/77529#discussion_r284017463), I could help implement it. 
cc @bsalamat